<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddAction</class>
 <widget class="QWidget" name="AddAction">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Voice Action</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>301</width>
     <height>391</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>15</number>
    </property>
    <property name="topMargin">
     <number>8</number>
    </property>
    <property name="rightMargin">
     <number>15</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item row="2" column="0" colspan="3">
     <widget class="QLineEdit" name="lineEdit_name"/>
    </item>
    <item row="6" column="0" colspan="3">
     <widget class="QLineEdit" name="lineEdit_command"/>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="button_path">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="3">
     <widget class="QListView" name="listView"/>
    </item>
    <item row="7" column="0">
     <widget class="QPushButton" name="button_add">
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QPushButton" name="button_edit">
      <property name="text">
       <string>Edit</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Command name:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="3">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Associative words:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="3">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Path to file:</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QPushButton" name="button_delete">
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QLineEdit" name="lineEdit_path"/>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>listView</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>AddAction</receiver>
   <slot>chooseCom(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listView</sender>
   <signal>entered(QModelIndex)</signal>
   <receiver>AddAction</receiver>
   <slot>changeValue(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_path</sender>
   <signal>clicked()</signal>
   <receiver>AddAction</receiver>
   <slot>getPathDial()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>268</x>
     <y>315</y>
    </hint>
    <hint type="destinationlabel">
     <x>175</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_add</sender>
   <signal>clicked()</signal>
   <receiver>AddAction</receiver>
   <slot>addAction_onClickButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>389</y>
    </hint>
    <hint type="destinationlabel">
     <x>180</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_delete</sender>
   <signal>clicked()</signal>
   <receiver>AddAction</receiver>
   <slot>deleteAction_onClickButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>170</x>
     <y>389</y>
    </hint>
    <hint type="destinationlabel">
     <x>180</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_edit</sender>
   <signal>clicked()</signal>
   <receiver>AddAction</receiver>
   <slot>editAction_onClickButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>262</x>
     <y>389</y>
    </hint>
    <hint type="destinationlabel">
     <x>180</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>chooseCom(QModelIndex)</slot>
  <slot>changeValue(QModelIndex)</slot>
  <slot>getPathDial()</slot>
  <slot>addAction_onClickButton()</slot>
  <slot>deleteAction_onClickButton()</slot>
  <slot>editAction_onClickButton()</slot>
 </slots>
</ui>
